src/en/actions.md

```diff
@@ -102,10 +104,17 @@ application). So if there were two git units you can also do:
 juju run-action git/0 git/1 add-repo repo=myproject
 ```
 
-When running short-lived actions from the command line, it is more convenient to
-add the `--wait` option to this command. This causes the Juju client to wait for
-the action to run, and then return the results and other information in YAML
-format.
+To run an action on an application leader append the string "/leader" to the
```

Comment:
ah, I guess it's in this section. I read the diff a bit wrong. Thanks for the update.